internal/core/api.rs

```diff
@@ -397,7 +394,9 @@ impl Window {
 
     /// This function issues a request to the windowing system to redraw the contents of the window.
     pub fn request_redraw(&self) {
-        self.0.window_adapter().request_redraw();
+        if let Some(x) = self.0.window_adapter().internal(crate::InternalToken) {
+            x.request_redraw()
```

Comment:
I think that you could make `request_redraw` also public at the same time. After all `request_redraw` as name is also in our public `slint::Window` API.